File: JANUARY/26-01-2026/1200.java
class Solution {
    public List<List<Integer>> minimumAbsDifference(int[] arr) {
        Arrays.sort(arr);
        int mn = Integer.MAX_VALUE;
        for (int i = 1; i < arr.length; i++)
            mn = Math.min(mn, arr[i] - arr[i - 1]);

        List<List<Integer>> res = new ArrayList<>();
        for (int i = 1; i < arr.length; i++)
            if (arr[i] - arr[i - 1] == mn)
                res.add(Arrays.asList(arr[i - 1], arr[i]));
        return res;
    }
}

File: JANUARY/26-01-2026/1200.dart
class Solution {
  List<List<int>> minimumAbsDifference(List<int> arr) {
    arr.sort();
    int mn = 1 << 30;
    for (int i = 1; i < arr.length; i++)
      mn = mn < arr[i] - arr[i - 1] ? mn : arr[i] - arr[i - 1];

    List<List<int>> res = [];
    for (int i = 1; i < arr.length; i++)
      if (arr[i] - arr[i - 1] == mn)
        res.add([arr[i - 1], arr[i]]);
    return res;
  }
}